tests/unit/test_control_connection.py

```
@@ -312,6 +327,91 @@ def test_wait_for_schema_agreement_none_timeout(self):
         cc._time = self.time
         assert cc._wait_for_schema_agreement()
 
+    def test_topology_queries_use_paging(self):
+        self.control_connection.refresh_node_list_and_token_map()
+        assert self.connection.wait_for_response.called
+        calls = self.connection.wait_for_response.call_args_list
+        for call in calls:
```

Ruff (0.16.0)

[error] Import `call` from line 18 shadowed by loop variable

(F402)